addon/components/loading-spinner.js

```diff
@@ -3,6 +3,10 @@ import layout from '../templates/components/loading-spinner';
 
 export default Component.extend({
   layout,
+
+  classNames: ['loading-spinner'],
   tagName: 'span',
-  classNames: ['loading-spinner']
+
+  class: '',
```

Comment:
This overloads this component too much and makes it harder to replace. Just stick with the regular spinner for now and don't worry about making it orange.